### DISCUSSION TITLE
new user seeking to create project

### DISCUSSION THREAD

**Discussion by SB2020-eye** [biigle/core#361] [General]
Hi. I would like to try biigle for a hobby project I am working on. I created an account. What is the process for being able to create a project?

(I see the statement, "You can request a project admin to add you to a project." Assuming I understand this correctly, I don't really want to be added to anyone else's project. I want to try it out on the small, personal, solo project I am working on involving images from a medieval manuscript.)

**Comment by mzur** [comment]
We have to approve your account (which I just did) before you can create your own projects.

**Comment by SB2020-eye** [comment]
Thank you so much! I apologize if I was being too hasty. Maybe an added message alongside the existing one ("You can request a project admin to add you to a project") would be helpful. Something telling new, independent users what exactly to do (eg, "Please wait for a biigle admin to approve your account", or "Please send an email to <email address> describing your use case in order for your account to be approved", or even "We are not accepting new users at this time" if that's the case.)

Thanks again!

**Reply by mzur** [reply]
You should have seen the one-time message "Welcome to BIIGLE! Take a look around and make yourself at home. You will be able to create new projects or label trees once an admin has accepted your sign up." on your dashboard.

**Comment by SB2020-eye** [comment]
Yes. Lol. I failed to mention that I did see a message at the very outset. Then it went away. (Perhaps I clicked away, or clicked something before reading it thoroughly.) Anyways, once it was gone, I couldn't recall what exactly it said. So I only had that other message in front of me -- about being admitted to an existing project. I did do a search through issues and the documentation before posting here, but couldn't find anything else relevant. But maybe that was just not using the best search criteria.

Anyways, the main thing is _thank you_. And anything beyond the one-time message that goes away would only be for people like me that click too eagerly and/or don't read everything carefully the first time. 😄 

All the best!

**Comment by SB2020-eye** [comment]
Sorry for another noob question: how do I annotate an image on my computer? I hit "create new volume", but I see one is supposed to enter an https address. I don't have any kind of server or anything.

**Reply by mzur** [reply]
Please have a look at [remote volumes](https://biigle.de/manual/tutorials/volumes/remote-volumes) in the manual. As BIIGLE is web-based, you can't annotate images that are stored locally on your computer. If you can't set up a web server for a remote volume, we offer a few GB of storage per user on our servers. Please write an email to <info@biigle.de> in this case.
